test/acqusition.test.mjs

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+import assert from 'assert';
+import { describe, it } from 'node:test';
+import { classifyAcquisition } from '../src/acquisition.mjs';
+
+describe('classifyAcquisition', () => {
+  const testCases = [
+    { input: 'google', expected: 'paid:search:google', only: true },
+    { input: 'facebook', expected: ':social:facebook' },
+    { input: 'fb', expected: ':social:facebook' },
+    { input: 'bing', expected: 'paid:search:bing' },
+    { input: 'ig', expected: ':social:instagram' },
+    { input: 'meta', expected: ':social:facebook' },
+    { input: 'youtube', expected: ':video:youtube' },
+    { input: 'linkedin', expected: ':social:linkedin' },
+    { input: 'gmb', expected: '' },
+    { input: 'tiktok', expected: 'paid:video:tiktok', only: true },
+    { input: 'newsletter', expected: 'owned:email' },
+    { input: 'instagram', expected: ':social:instagram' },
+    { input: 'dv360', expected: '' },
+    { input: 'email', expected: 'owned:email' },
+    { input: 'programmatic', expected: 'paid:display' },
+    { input: 'googleads', expected: 'paid:search:google', only: true },
+    { input: 'sfmc', expected: '' },
+    { input: 'google-ads', expected: 'paid:search:google', only: true },
+    { input: 'an', expected: '' },
+    { input: 'marketo', expected: 'owned:email:marketo', only: true },
+    { input: 'hs_email', expected: 'owned:email' },
+    { input: 'social', expected: ':social', only: true },
+    { input: 'microsoft', expected: 'paid:display:microsoft', only: true },
+    { input: 'adwords', expected: 'paid:search:google', only: true },
+    { input: 'pinterest', expected: ':social:pinterest' },
+    { input: 'gdn', expected: '' },
+    { input: 'facebook-instagram', expected: ':social:instagram', only: true },
+    { input: 'twitter', expected: ':social:x' },
+    { input: 'snapchat', expected: ':social:snapchat' },
+    { input: 'eloqua', expected: 'owned:email:eloqua', only: true },
+    { input: 'yahoo', expected: 'paid:search:yahoo', only: true },
+    { input: 'teads', expected: 'paid::teads' },
+    { input: 'criteo', expected: 'paid:display:criteo', only: true },
+    { input: 'acs', expected: '' },
+    { input: 'taboola', expected: 'paid:display:taboola', only: true },
+    { input: 'kaufland marketing', expected: '' },
+    { input: 'search', expected: 'paid:search', only: true },
```

Comment:
Why paid? This only tells us it's search, no?

Reply:
Organic search results never have UTM parameters.

Reply:
Good point!